apps/src/studio/studio.js

```diff
@@ -1826,6 +1882,55 @@ function spriteTotalFrames (index) {
     sprite.frameCounts.turns + sprite.frameCounts.emotions;
 }
 
+function cellId(prefix, row, col) {
+  return prefix + '_' + row + '_' + col;
+}
+
+Studio.drawWallTile = function (svg, row, col) {
+
+  // Placeholder implementation: just drawing boxes with X's in them for now:
+
```

Comment:
cool.  we will definitely want them to pick randomly or psuedo-randomly from a larger set of wall images longer term.